django/contrib/auth/__init__.py

```diff
@@ -172,7 +172,7 @@ def get_user_model():
     Returns the User model that is active in this project.
```

Comment:
For the commit message, add "Fixed #25966" (and maybe "Refs #" to at least some of the other commits?).